otter/test/convergence/test_steps.py

```diff
@@ -609,8 +611,7 @@ def test_remove_nodes_from_clb_success_failures(self):
         :obj:`AddNodesToCLB` succeeds if the CLB is not in existence (has been
         deleted or is not found).
         """
-        successes = (CLBDeletedError(lb_id=u"12345"),
-                     NoSuchCLBError(lb_id=u"12345"))
+        successes = [CLBNotFoundError(lb_id=u'12345')]
```

Comment:
Non-blocking - similar to @manishtomar's comment above, do we want to ensure that the subclasses are also counted as success as well as the superclass?